workbench/src/main/setupAddRemovePlugin.js

```diff
@@ -27,7 +27,8 @@ const logger = getLogger(__filename.split('/').slice(-1)[0]);
  */
 function spawnWithLogging(cmd, args, options) {
   logger.info(cmd, args);
-  const cmdProcess = spawn(cmd, args, { ...options, windowsHide: true });
+  const cmdProcess = spawn(
+    cmd, args, { ...options, shell: true, windowsHide: true });
```

Comment:
Suggest mentioning this in the docstring

Reply:
Added to docstring in https://github.com/natcap/invest/pull/1688/commits/276632db06803c29b00d469f65ac2df0b95f99e7